src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -615,8 +602,10 @@ private void processLogsAndUpload() throws InterruptedException {
 
                     allFiles.forEach(file -> {
                         long startPosition = 0;
+                        Optional fileHash = Optional.ofNullable(calculateHashForLogFile(file));
```

Comment:
Shouldn't need to wrap it around a `Optional.ofNullable`. `calculateHashForLogFile` already returns an optional